composer.json

```diff
@@ -93,18 +92,18 @@
         "symfony/debug-bundle": "^3.0",
         "symfony/dom-crawler": "^3.0",
         "symfony/monolog-bundle": "^3.0",
-        "symfony/phpunit-bridge": "^3.0",
+        "symfony/monolog-bridge": "^3.0",
+        "symfony/phpunit-bridge": "^3.0.4",
```

Comment:
needed as 3.0.4 else the WEAK flag for deprecations don't work and it throws an error